### PR TITLE
Use int type for SizeInBytes in file summary response

### LINKED ISSUE #11: Unmarshal Bug in Dataset.Retrieve
```
retrieveResp, err := dw.Dataset.Retrieve("associatedpress", "johns-hopkins-coronavirus-case-tracker")
if err != nil {
    fmt.Fprintln(os.Stderr, "Dataset.Retrieve() returned an error:", err)
    os.Exit(1)
}
```
//returns:
//Dataset.Retrieve() returned an error: json: cannot unmarshal number into Go struct field FileSummaryResponse.files.sizeInBytes of type string

### PR DESCRIPTION
**Pull request by chrisbodhi**
Closes #11

### CODE REVIEW

**Review by jmoney — CHANGES_REQUESTED** [review]
Use int64

**Review by jmoney — CHANGES_REQUESTED** [review]
Bonus points if you use unint, as you can't have negative bytes

File: dwapi/models.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ type FileSummaryResponse struct {
 	Description string             `json:"description,omitempty"`
 	Labels      []string           `json:"labels,omitempty"`
 	Name        string             `json:"name"`
-	SizeInBytes string             `json:"sizeInBytes,omitempty"`
+	SizeInBytes uint               `json:"sizeInBytes,omitempty"`
```
Comment (jmoney) [inline dwapi/models.go:124]: uint64

**Review by jmoney — CHANGES_REQUESTED** [review]

**Review by jmoney — DISMISSED** [review]

**Comment by chrisbodhi** [comment]
I was too cool, and rebased instead of merging. 😑

**Review by jmoney — APPROVED** [review]

**Pull request merged**